tests/tools/conftest.py

```diff
@@ -0,0 +1,19 @@
+"""Conftest."""
+
+from typing import List
+import pytest
+
+from llama_index.readers.schema.base import Document
+
+
+@pytest.fixture
+def documents() -> List[Document]:
```

Comment:
Why are we duplicating this. This already exists under `tests/conftest.py?

Reply:
hmm mentioned offline too but i only see it in `tests/indices/conftest.py`